src/tasks/twitchRefreshSubscriptions.ts

```diff
@@ -31,16 +32,13 @@ export default class extends Task {
 		for (const subscription of allSubscriptions) {
 
 			// If the subscription has an expiry date that's before the current date then queue that subscription for refreshing
-			if (subscription.expires_at < currentDate) {
+			if (subscription.expiresAt.getTime() < currentDate) {
 
 				// Increase the updated subcriptions ticker by 1
 				updatedSubsTicker++;
 
 				// Queue the updating by pushing the promise into the promises array
-				promises.push(
-					this.client.twitch.subscriptionsStreamHandle(subscription.id, TwitchHooksAction.Subscribe).catch(error => this.client.emit(Events.Wtf, error)),
-					this.client.queries.upsertTwitchStreamSubscription(subscription.id).catch(error => this.client.emit(Events.Wtf, error))
```

Comment:
This needs to be restored. `subscriptionStreamHandle` only resubs on Twitch but it doesn't update the entry in the database.

Reply:
But the entry is already there, as you see in a few lines above where we fetch it.

Reply:
Yes and the expireAt has to be updated to the new date otherwise it'll think it has to refresh it for another 10 days the next time the task runs

Reply:
They should be updated using a TypeORM transaction, iirc using `createQueryRunner()` (so we can use `Promise.all`).
- The previous code was doing an UPSERT operation for each entry even though they already existed, here we only need to update the field and do a batch update.
- The other way to do it is by using `update()` which takes a condition (can be used to select all streamer IDs) and the new date for the `refreshAt`, this way we do a single query as opposed to making multiple queries within a transaction.